tasmota/xdrv_08_serial_bridge.ino

```diff
@@ -65,7 +65,7 @@ void SerialBridgeInput(void) {
   while (SerialBridgeSerial->available()) {
     yield();
     uint8_t serial_in_byte = SerialBridgeSerial->read();
-
+    serial_bridge_raw = Settings->serial_delimiter == 255;
```

Comment:
As serial_delimiter is default 255 this will force any received data in raw mode by default. This breaks current assumptions.

It's better to test on 254.